### ISSUE TITLE
No game while it actually exists 

### ISSUE THREAD

**Issue by Xeononyum** [love2d/love-android#253]
I installed love and directory sdcard/Android/data/org.love2d.android didnt existed
so i created it, and then created files/games/lovegame and main.lua file but when i starting love for android it says no game

**Comment by Xeononyum** [comment]
Version 11.3 works fine even with game in /sdcard/lovegame (android 11)

**Comment by MikuAuahDark** [comment]
If you're using 11.4, then `/sdcard/Android/data/org.love2d.android/files/games/lovegame` should work. If you're using 11.3, then `/sdcard/lovegame` should work.

**Comment by Xeononyum** [comment]
https://github.com/love2d/love-android/assets/90855565/deff6b52-d5b4-4b36-80c8-341d22544d57

It doesnt (11.4)

**Comment by Xeononyum** [comment]
And yes org.love2d.android didnt existed

**Comment by HarrievG** [comment]
For any future readers.
There is no way on unrooted devices to use 11.3 > without having a shitton of permission pain.
Just downgrade to 11.3 and you will be able to edit lua files on your device in the good old /sdcard/lovegame.

_The core problem here is that you are not able to copy files to or edit files in_ **/sdcard/Android/data/org.love2d.android/files/games/lovegame**
 on an unrooted device
 
 With ADB you are able to create and copy files to that location. even unrooted adb is able to.
 
 TLDR
 Love2d 11.3 > forces the usage of /sdcard/Android/data/org.love2d.android/files/games/lovegame, which is a protected location.
 If you want to be able to edit lua files and run directly on your android device use 11.3.
